dali/test/python/test_operator_cast.py

```diff
@@ -102,8 +101,8 @@ def generate(rng, ndim: int, batch_size: int, in_dtype: np.dtype, out_dtype: np.
             hi = min(np.finfo(in_dtype).max, hi)
 
     max_size = 100000 // batch_size
-    out = [rng.uniform(lo, hi, size=random_shape(rng, ndim, max_size)).astype(in_dtype) for _ in range(batch_size)]
-    out = replace_with_empty_volumes(rng, out, empty_volume_policy)
```

Comment:
Why removing that
`out = replace_with_empty_volumes(rng, out, empty_volume_policy)`?

Reply:
Must've happened by mistake. Reverted